src/index.js

```diff
@@ -203,6 +221,8 @@ class Dropzone extends React.Component {
       inputAttributes.name = name;
     }
 
+    if (this.props.global) return null;
```

Comment:
I'd have to pull this down and test it... but I don't think you want to necessarily return null here. If global (or whatever the property is called) is specified I'd still like to render the dropzone div with the input. If preventing the output is a requirement for you, I'd suggest separating that into another property. I can certainly see a use case for having a drop area with content that changes whenever a file is dragged over the entire window.